src/containers/NFT/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import NoMatch from '../NoMatch';
+import NFTHeader from './NFTHeader';
+import './styles.css';
+import { analytics, ANALYTIC_TYPES, NOT_FOUND, BAD_REQUEST } from '../shared/utils';
+
+const ERROR_MESSAGES = {};
+ERROR_MESSAGES[NOT_FOUND] = {
+  title: 'assets.no_nfts_message',
+  hints: ['check_nft_id'],
+};
+ERROR_MESSAGES[BAD_REQUEST] = {
+  title: 'invalid_xrpl_address',
+  hints: ['check_nft_id'],
+};
+ERROR_MESSAGES.default = {
+  title: 'generic_error',
+  hints: ['not_your_fault'],
+};
```

Comment:
why not build this as a literal in the first place?

```javascript
const ERROR_MESSAGES = {
  [NOT_FOUND]: {
    title: 'assest.no_nfts_message',
    hints: ['check_nft_id'],
  },
  ...,
};
```

Reply:
oops didnt know it. fixed!